pulpcore/pulpcore/app/models/publication.py

```diff
@@ -20,6 +42,68 @@ class Publication(Model):
 
     repository_version = models.ForeignKey('RepositoryVersion', on_delete=models.CASCADE)
 
+    complete = False  # placeholder
+
+    @classmethod
+    def create(cls, repository_version, publisher):
+        """
+        Create a publication.
+
+        This should be used to create a publication.  Using Publication() directly
+        is highly discouraged.
+
+        Args:
+            repository_version (pulpcore.app.models.RepositoryVersion): The repository
+                version to be published.
+            publisher (pulpcore.app.models.Publisher): The publisher used
+                to create the publication.
+
+        Returns:
+            pulpcore.app.models.Publication: self
+
+        Notes:
+            The created publication with complete=False.
```

Comment:
Would you move this note to the returns line? Something like
Returns:
     A created Publication in an incomplete state.

Reply:
👍